src/Driver/PgSQL/Statement.php

```diff
@@ -151,10 +157,28 @@ public function execute($params = null): Result
             switch ($this->parameterTypes[$parameter]) {
                 case ParameterType::BINARY:
                 case ParameterType::LARGE_OBJECT:
+                    $isResource = $value !== null && is_resource($value);
```

Comment:
The `null` check is redundant.

```suggestion
                    $isResource = is_resource($value);
```